rust/src/arrow/linalg.rs

```diff
@@ -177,6 +181,7 @@ impl MatrixView {
     }
 
     /// Dot multiply
+    #[cfg(feature = "opq")]
```

Comment:
let's file a ticket to implement dot w/o Blas?

Reply:
Should I move this into its own module outside of `linalg.rs` as well?